tests/Modules/ASayModule.Tests/ASayControllerTest.cs

```diff
@@ -0,0 +1,39 @@
+using EvoSC.Modules.Official.ASayModule.Controllers;
+using EvoSC.Modules.Official.ASayModule.Interfaces;
+using Moq;
+
+namespace EvoSC.Modules.Official.ASayModule.Tests;
+
+public class ASayControllerTest
```

Comment:
You can also inherit the `CommandInteractionControllerTestBase` to make this even cleaner and be consistent with other controller tests.

Example: https://github.com/EvoEsports/EvoSC-sharp/blob/master/tests/Modules/Player.Tests/PlayerCommandsControllerTests.cs

Reply:
done